ekingan/to-do/awesome-todo

```diff
@@ -0,0 +1 @@
+Subproject commit 98c06e4837df47326711d82573721bd372d6df30
```

Comment:
seems like you copied the awesome-todo in here.  It should probably be in your ~/dev/ dir